flow-service/src/config/logging-config-extractor.ts

```
@@ -0,0 +1,159 @@
+    const logFilePath = await getChannelProperty('fsvc:hasFileChannel', 'fsvc:logFilePath');
+    const logRetentionDays = await getChannelProperty('fsvc:hasFileChannel', 'fsvc:logRetentionDays');
+    const logMaxFiles = await getChannelProperty('fsvc:hasFileChannel', 'fsvc:logMaxFiles');
+    const logMaxFileSize = await getChannelProperty('fsvc:hasFileChannel', 'fsvc:logMaxFileSize');
+    const logRotationInterval = await getChannelProperty('fsvc:hasFileChannel', 'fsvc:logRotationInterval') as 'daily' | 'weekly' | 'monthly' | 'size-based' | undefined;
+
+    return {
+      logChannelEnabled: fileLoggingConfig.enabled,
+      logLevel: (fileLoggingConfig.level as LogLevel) || 'warn',
+      logFormat: logFormat || 'json',
+      logFilePath: logFilePath || './logs/flow-service.log',
+      logRetentionDays: logRetentionDays ? parseInt(logRetentionDays) : 30,
```

ESLint

[error] Unquoted property 'logRetentionDays' found.

(quote-props)